drivers/ethernet/eth_mcux.c

```diff
@@ -1395,7 +1395,7 @@ static void eth_mcux_err_isr(const struct device *dev)
 									\
 	ETH_NET_DEVICE_DT_INST_DEFINE(n,					\
 			    eth_init,					\
-			    ETH_MCUX_PM_FUNC,				\
+			    eth_mcux_device_pm_action,		\
```

Comment:
This change looks wrong. You need to keep `ETH_MCUX_PM_FUNC`, which is conditionally defined as `eth_mcux_device_pm_action` or `NULL`

Reply:
fixed, thanks for spotting this one